notebook/static/tree/js/notebooklist.js

```diff
@@ -697,6 +707,73 @@ define([
         });
     };
 
+    NotebookList.prototype.move_selected = function() {
+        // TODO(nhdaly): Support moving multiple items at once.
+        if (this.selected.length !== 1){
+            return;
+        }
+
+        var that = this;
```

Comment:
Done! Thanks. (Extremely new to Javascript; why do we even create the `that` variable? I was just following the convention of the surrounding functions.)